test-bench-utils/lib/sinks/crypto.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const crypto = require('crypto');
+
+/**
+ * @param {string} not used but need to keep signature for abstracted helpers
+ * @param {Object} opts
+ * @param {boolean=} opts.safe are we calling the sink safely?
+ * @param {boolean=} opts.noop are we calling the sink as a noop?
+ */
+module.exports['crypto-bad-mac'] = async function exec(
+  input,
+  { safe = false, noop = false } = {}
+) {
```

Comment:
We could use the `_param` pattern here to indicate an unused argument.

```suggestion
/**
 * @param {string} _input not used but need to keep signature for abstracted helpers
 * @param {Object} opts
 * @param {boolean=} opts.safe are we calling the sink safely?
 * @param {boolean=} opts.noop are we calling the sink as a noop?
 */
module.exports['crypto-bad-mac'] = async function exec(
  _input,
  { safe = false, noop = false } = {}
) {
```